bittide-instances/src/Bittide/Instances/Hitl/FullMeshHwCc.hs

```diff
@@ -230,7 +222,7 @@ fullMeshHwTest ::
   , "ALL_UP" ::: Signal Basic125 Bool
   , "ALL_STABLE"   ::: Signal Basic125 Bool
   )
-fullMeshHwTest refClk sysClk testRst IlaControl{..} rxns rxps miso =
+fullMeshHwTest refClk sysClk IlaControl{syncRst = rst, ..} rxns rxps miso =
```

Comment:
Personally I find this a bit confusing. Why not switch `rst` and `syncRst` ?


Then you'll just have:
```hs
fullMeshHwTest refClk sysClk IlaControl{..} rxns rxps miso = _
 where
   rst = syncRst `orReset` otherReset
```

Reply:
It's because I like to pass `syncRst` (with that variable name) to `callistoClockControlWithIla` later, but the resets for `fullMeshHwTest` and `callistoClockControlWithIla` are different although they share the same name. The latter reset is an extension of the prior one, which is why I prefer to use the "overwriting semantics" via

```haskell
syncRst = rst `orReset` (unsafeFromActiveLow (fmap not spiErr))
```
which is intentionally placed right below to be easy to catch.

This is a general issue all over the place, as we extend the reset lines multiple times to incorporate more and more failure indicators. Maybe it's worth to think about a more general solution for this as well.